### PR TITLE
Added Tori and Simon to CAST and Fuse sites

### PR DESCRIPTION
**Pull request by eghale**

### CODE REVIEW

File: data/teams.yml [hunk 1]
```diff
@@ -75,29 +79,32 @@ teams:
...
+  - On Fuse, Women’s Aid has created a new interactive online service for girls aged 14-18 to answer their questions about relationships. They then receive advice and support from experts, young survivors of abuse and their peers, to emphasise that they are not alone.
+  - The service has been developed in consultation with over 150 young women across the country, and is currently being tested ahead of an Alpha launch.
+  - <em>“We are indebted to CAST for their excellent support and guidance at every step of this project. The Fuse programme has given us the tools and confidence to embed user-centred design principles not only in this project but in other initiatives across the organisation. We are are proud to be collaborating with young survivors of abuse on the development of this new service, together we want to make sure that no one else has to go through what they experienced.”<em>
```
Comment (ivan-unfolds) [inline data/teams.yml]: @eghale This `<em>` is not closed so it is causing the Clore Social image to go funny.

**Review by andrewMacmurray — COMMENTED** [review]
This looks mostly good. A general point I'd make is that when you name image files it's best to use dashes instead of spaces and use lowercase (so `CAST Logo XL.png` should be `cast-logo-xl.png`). Other than the `<em>` change it's good to go 👍

File: data/teams.yml (hunk 1 above)
Comment (ivan-unfolds) [reply data/teams.yml]: Lol it is actually making everything underneath this italicised.

**Comment by eghale** [comment]
@andrewMacmurray @ivanmauricio So can this be merged now too? Can everything be merged?

**Review by andrewMacmurray — APPROVED** [review]
👍

**Pull request merged**